scripts/user_control.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+import rospy
+import roslib
+import time
+import cv2 as cv2
+from pynput import keyboard
+import sys
+import atexit
+import curses
+
+from std_msgs.msg import Bool
+from sensor_msgs.msg import Image
+from geometry_msgs.msg import Twist
+from geometry_msgs.msg import Wrench
+from cv_bridge import CvBridge, CvBridgeError
+from uuv_gazebo_ros_plugins_msgs.msg import FloatStamped
+from sensor_msgs.msg import Image
+import numpy as np
+import matplotlib.pyplot as plt
+
+class UserControl:
+    def __init__(self):
+        self.gain = [30, 30, 60, 20, 20, 8]       
+        self.pub = rospy.Publisher("/manta/thruster_manager/input", Wrench, queue_size=1)
```

Comment:
If you move this node, make sure to change this publisher to use a relative instead of an absolute path.